app/v1/controllers/requestParsers/validators/CreateAmendOtherCgtValidator.scala

```diff
@@ -65,6 +65,26 @@ class CreateAmendOtherCgtValidator @Inject()(implicit appConfig: AppConfig)
     standardValidation ++ emptyValidation
   }
 
+  private def oneMinimumRequiredFieldValidator: CreateAmendOtherCgtRawData => List[List[MtdError]] = { data =>
+    val requestBodyData = data.body.json.as[CreateAmendOtherCgtRequestBody]
+
+    List(Validator.flattenErrors(
+      List(
+        requestBodyData.nonStandardGains.map {
+          case data => oneOfThreeGainsSuppliedValidation(data)
+        }.getOrElse(NoValidationErrors)
```

Comment:
this should probably be a `fold` statement, or something like
```suggestion
        requestBodyData.nonStandardGains.map(oneOfThreeGainsSuppliedValidation(_)).getOrElse(NoValidationErrors)
```